filters/apiusagemonitoring/spec.go

```diff
@@ -32,6 +35,7 @@ func NewApiUsageMonitoring(
 	realmKeys string,
 	clientKeys string,
 	realmsTrackingPattern string,
+	tracer opentracing.Tracer,
```

Comment:
We could keep backwards compatibility, if we created a new function, e.g. `NewApiUsageMonitoringWithTracing`, and the current one called the new with the NopTracer. Even if there's a low chance that somebody uses this function directly, it's nicer not to assume that nobody does.